ffi-cdecl/include/mxcfb-remarkable.h

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
Now that I've finished bothering you with my pesky tests, I'd recommend updating to the latest FBInk copy ;).

Reply:
Good plan, done